api/v2/models/bouquets/bouquets_schema.py

```diff
@@ -5,5 +5,4 @@
 class BouquetsSchema(ma.Schema):
     class Meta:
         model = BouquetsModel
-        fields = ("id", "bouquet_id", "api_key1",  "api_key2"
-                  "auth_credentials", "bouquet_name", "should_refresh")
+        fields = ("id", "bouquet_id", "api_key1",  "api_key2", "auth_credentials", "bouquet_name", "should_refresh")
```

Comment:
this should_refresh field name indicates what exactly?? what is its purpose, I am totally confused @kodek-sleuth

Reply:
Imagine if in the future we were to supply a job(celery) to this function to always refresh and get channels, this would be the basis on which we would base from